docs/workers.md

```diff
@@ -185,6 +185,9 @@ reverse-proxy configuration.
 The `^/_matrix/federation/v1/send/` endpoint must only be handled by a single
 instance.
 
+Note that the `worker_listeners.resources.name` needs to be set to `federation`
+for this worker.
```

Comment:
Does `name` work? I thought it was `worker_listeners.resources.names`, which was then a list.

I'd write something like:

Note that `federation` must be added to the listener resources in the worker config:

```
    worker_app: synapse.app.federation_reader
    ...
    worker_listeners:
     - type: http
       port: <port>
       resources:
         - names:
           - federation
```

Reply:
Yeah that is better, thanks! Fixed